Header header
geometry_msgs/Pose start
geometry_msgs/Pose goal
int64 turning_radius
int64 planning_time
---
Header header
std_msgs/Bool success
geometry_msgs/PoseArray path
